vec-redis/setup-redis.sh

```diff
@@ -41,7 +41,6 @@ MODULEDIR="$tmpdir/lib"
  --protected-mode no \
  --daemonize no \
  --loadmodule "${MODULEDIR}/redisearch.so" ${REDISEARCH_ARGS} \
- --loadmodule "${MODULEDIR}/redisgears.so" v8-plugin-path "${MODULEDIR}/libredisgears_v8_plugin.so"  ${REDISGEARS_ARGS} \
  "$@") &
```

Comment:
are you sure you can't do this?

```suggestion
 --loadmodule "${MODULEDIR}/redisgears.so" v8-plugin-path "${MODULEDIR}/libredisgears_v8_plugin.so" \
 "$@") &
```

actually also is it still safe to put "$@" here? how does `--loadmodule` know which args are for the module and which are for `redis-stack`?

Reply:
so we don't need `redisgears` for vector db only `redisearch` is needed, and `--loadmodule` seems to take only the required arguments (from whatever I've tested locally) and all other arguments are passed down to `redis-server` hence `setup-redis.sh --port 8001` works